iroh-net-report/src/lib.rs

```diff
@@ -86,6 +90,7 @@ pub struct Report {
     /// public IP address (on IPv4).
     pub hair_pinning: Option<bool>,
     /// Probe indicating the presence of port mapping protocols on the LAN.
+    #[cfg(not(wasm_browser))]
```

Comment:
given how many things are cfged out, maybe it makes sense to split up the report in a couple of scoped structs

Reply:
Actually, I just realized, this is mostly just options, so this is fine.

Reply:
Yeah - I've opted to keep the cfgs in the `Options` struct now.